### PR TITLE
Removal of unused files (#188)

### PR DESCRIPTION
**Pull request by UtiluMark**

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+Next version
+==================
+
+  * Extension: Removed the unused CSS files
+
```
Comment (whimboo) [inline History.md]: No need to update the history.md manually for each change. We do this via git itself once a new version is about to be released.

File: extension/chrome/skin/session/session.css [hunk 2]
```diff
@@ -1,7 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-treechildren::-moz-tree-checkbox(checked) {
-  list-style-image: url("chrome://nightly/skin/cbox-check.gif");
```
Comment (whimboo) [inline extension/chrome/skin/session/session.css]: Looks like this image is only used in this selector and can also be removed.

File: History.md (hunk 1 above)
Comment (UtiluMark) [inline History.md]: That's fine, I've reverted History.md.

File: extension/chrome/skin/session/session.css (hunk 2 above)
Comment (UtiluMark) [inline extension/chrome/skin/session/session.css]: That's right, the cbox-check.gif image isn't referenced anywhere else, so I've removed it as well.

**Comment by UtiluMark** [comment]
This concludes the removal of the unused files and the code clean-up of all the files.

**Comment by UtiluMark** [comment]
Could you merge this pull request?

**Comment by whimboo** [comment]
@UtiluMark I wanted to keep this PR clean and only covering the code removals. All other commits should actually be on a different PR. Can you please fix that?

**Comment by whimboo** [comment]
Btw. this PR will fix a small part of issue #188.

**Comment by UtiluMark** [comment]
@whimboo I've reverted the non-removal/cleanup commits.

**Comment by whimboo** [comment]
There are still 24 changed files. What I want is the version of the PR you had at first when I reviewed it. I think it would be good if you better get rid of all those extra commits.

**Comment by UtiluMark** [comment]
@whimboo OK, I've reverted everything except the removal of the three unused files.

**Comment by whimboo** [comment]
That looks fine now. Thanks. I will get the commits squashed and merged to master.

**Comment by whimboo** [comment]
PR got merged to master as https://github.com/mozilla/nightlytt/commit/eff8f9e3eac6dcaa6f38c9341836a0bba5966209.

Now you can create a new PR with all the coding style changes. :) I would suggest that you make use of a new branch in your repository so that you don't destroy your master branch. Thanks again.

**Comment by UtiluMark** [comment]
@whimboo I've created PR #189 for one commit https://github.com/UtiluMark/nightlytt/commit/878c7885c6fa33260995aeeba97c163ae70c3884 Removed excessive white space before line endings and fixed indentation by always using spaces and never using tabs.

**Comment by whimboo** [comment]
Thank you for the other PR. Best would be to simply add a comment on the newly created PR. That one is actually closed. :)

**Pull request closed without merging**